lib/challenge.js

```diff
@@ -1,305 +1,327 @@
+ *
+ * Sample Output IN THE BROWSER:
+ *   - I found a 5. High five!
+ *   - I found a 10. High five!
+ */
+function prompt7() {
+  for (let i = 0; i <= 100; i++ ) {
+    if (i % 5 === 0) {
+      console.log("I have found a " + [i] + " High five!")
+    }
+  }
+}
+
+// COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
+prompt7()
+
+// 🌟 MAKE A COMMIT: "Complete prompt 7"
+
+/*
+ * Prompt 8:
+ *
+ * Create a variable called someNumber and assign it a random number between
+ * 0 and 100 (not inclusive). Hint: Use Math.random so that it generates a new number every time the code is run!
+ *
+ * Create a conditional that matches these requirements:
+ *   - if someNumber is less than 30, print "That's a small number!"
+ *   - if someNumber is between 30 and 60, print "The number is medium sized."
+ *   - if someNumber is greater than 60, print "We got a big one!"
+ */
+
+//  define someNumber here
+var someNumber = Math.floor(Math.random() * 99) +1;
+
+function prompt8() {
+  if (someNumber < 30) {
+  return "That's a small number!";
+} else if (someNumber >= 30 && someNumber <= 60) {
+  return "The number is medium sized.";
+      } else if (someNumber > 60) {
+        return "We got a big one!";
+  }
+  // print someNumber to the console in your function so you can see its value
+  console.log(someNumber);
+
+  // YOUR CODE HERE
+}
+
+// COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
+prompt8();
+
+// 🌟 MAKE A COMMIT: "Complete prompt 8"
+
+/*
+ * Prompt 9:
+ *
+ * Iterate over the starWars array printing each character's name and index
+ *
+ * i.e.:
+ *  0. Han
+ *  1. C3PO
+ *  2. R2D2
+ */
+
+const starWars = ["Han", "C3PO", "R2D2", "Luke", "Leia", "Anakin"];
+
+function prompt9() {
+  for (let i = 0; i < starWars.length; i++) {
+    console.log(starWars[i]);
+  }
+}
+
+// COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
+prompt9();
+
+// 🌟 MAKE A COMMIT: "Complete prompt 9"
+
+/*
+ * Prompt 10:
+ *
+ * Write a loop that pushes every even number between 0 and 100 into an array,
+ * then print the array.
+ */
+function prompt10() {
+  const myArray = [];
+  for (let i = 0; i <= 100; i ++) {
+    console.log(i);
+  return myArray;
+}
+
+// COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
+console.log(prompt10());
+
+// 🌟 MAKE A COMMIT: "Complete prompt 10"
+
+/*
+ * Prompt 11:
+ *
+ * Find the median number in the following nums array (the mid point), then
+ * console.log that number. The median number is the middle number in a SORTED list of numbers. Think through the steps you'll need to take to determine this number with code!
+ *
+ * Hint: What does Math.floor() do?
+ */
+
+let nums = [
+  14,
+  11,
+  16,
+  15,
+  13,
+  16,
+  15,
+  17,
+  19,
+  11,
+  12,
+  14,
+  19,
+  11,
+  15,
+  17,
+  11,
+  18,
+  12,
+  17,
+  12,
+  71,
+  18,
+  15,
+  12,
+];
+
+function prompt11() {
+  let median;
+  // YOUR CODE HERE
+
+  console.log(median);
+}

```

Comment:
This is a tricky one.  I recommend going to this site to see what a median is and then trying to write that logic in java script
https://www.mathsisfun.com/median.html

Break down the steps.  work on one step at a time...